framework/sandstone.h

```diff
@@ -76,6 +77,19 @@ extern "C" {
 /// followed by 0 or more arguments that provide data for the format string.  The message
 /// is only logged in debug builds.  This macro has no effect in release builds and generates
 /// no code.
+
+// skip categories
+typedef enum SkipCategory{
+    RESOURCE_UNAVAILABLE=65,
+    CPU_NOT_SUPPORTED=66,
+    DEVICE_NOT_FOUND=67,
+    DEVICE_NOT_CONFIGURED=68,
+    OTHERS=69,
+    RUNTIME_SKIP=70,
+}SkipCategory;
+
+#define log_skip(skip_category,...)           log_message_skip(thread_num, skip_category,SANDSTONE_LOG_SKIP __VA_ARGS__)
```

Comment:
Nitpicking: there should be a space before `SANDSTONE_LOG_SKIP`. Also, would be cool to have some comments here, similar to other log macros. Done nitpicking :)